lib/src/value/calculation.dart

```diff
@@ -158,6 +382,200 @@ class SassCalculation extends Value {
     return SassCalculation._("clamp", args);
   }
 
+  /// Creates a `pow()` calculation with the given [base] and [exponent].
+  ///
+  /// Each argument must be either a [SassNumber], a [SassCalculation], an
+  /// unquoted [SassString], a [CalculationOperation], or a
+  /// [CalculationInterpolation].
+  ///
+  /// This automatically simplifies the calculation, so it may return a
+  /// [SassNumber] rather than a [SassCalculation]. It throws an exception if it
+  /// can determine that the calculation will definitely produce invalid CSS.
+  ///
+  /// This may be passed fewer than two arguments, but only if one of the
+  /// arguments is an unquoted `var()` string.
+  static Value pow(Object base, Object? exponent) {
+    var args =
+        List<Object>.unmodifiable([base, if (exponent != null) exponent]);
+    _verifyLength(args, 2);
+    base = _simplify(base);
+    exponent = exponent.andThen(_simplify);
+    if (base is! SassNumber ||
+        exponent is! SassNumber ||
+        exponent.hasUnit('%') ||
+        base.hasUnit('%')) {
+      return SassCalculation._("pow", args);
+    }
+    _verifyCompatibleNumbers([base, exponent]);
```

Comment:
This doesn't guarantee that the operation can be performed at build time—it just ensures that the numbers don't have _definitely-incompatible_ units. Note that in the existing functions, we only call this before returning a `SassCalculation`, _not_ before running the actual operation.

This is something that should be covered by the tests.

Reply:
The `pow` function doesn't take unit numbers so this line shouldn't be here at all. Thanks for pointing this out!